### PR TITLE
Improve servername & database filters

### PR DESCRIPTION
**Pull request by ZuluPro**
Related to: 
- #206 Wrong DBBACKUP_CLEANUP_KEEP on multiple dbs
- #207 dbrestore --servername SERVERNAME doesn't work
- #208 mediarestore is missing --servername param
- #210 dbrestore --database DATABASE_NAME doesn't work

@adi-
Does it fix your issues ?

### CODE REVIEW

**Comment by adi-** [comment]
--servername in dbrestore doesn't work for me.

File: dbbackup/management/commands/dbrestore.py [hunk 1]
```diff
@@ -65,7 +65,8 @@ def _get_database(self, options):
 
     def _restore_backup(self):
         """Restore the specified database."""
-        input_filename, input_file = self._get_backup_file(database=self.database_name)
+        input_filename, input_file = self._get_backup_file(database=self.database_name,
+                                                           servername=self.servername)
         self.logger.info("Restoring backup for database: %s", self.database['NAME'])
```
Comment (adi-) [inline dbbackup/management/commands/dbrestore.py]: Would be nice to add servername too

**Comment by adi-** [comment]
```
./manage.py dbrestore --servername=test-www-dev --database=events
Finding latest backup
Reading file events-test-www-stg-2016-09-18-120601.psql
Reading file events-test-www-stg-2016-09-18-120601.psql
Restoring backup for server/database: events/test-www-dev
Restoring: events-test-www-stg-2016-09-18-120601.psql
```

**Comment by ZuluPro** [comment]
@adi- Thank for the feedback.
Could you try again ?

File: dbbackup/management/commands/dbrestore.py [hunk 2]
```diff
@@ -67,7 +67,8 @@ def _restore_backup(self):
         """Restore the specified database."""
         input_filename, input_file = self._get_backup_file(database=self.database_name,
                                                            servername=self.servername)
-        self.logger.info("Restoring backup for database: %s", self.database['NAME'])
+        self.logger.info("Restoring backup for database '%s' and server",
```
Comment (adi-) [inline dbbackup/management/commands/dbrestore.py]: `%s` is missing

Comment (adi-) [inline dbbackup/management/commands/dbrestore.py]: Besides that `--servername` is working.

**Comment by adi-** [comment]
@ZuluPro Anthony, I just wanted to thank you for such a quick response and added enhancement!

**Comment by ZuluPro** [comment]
@adi- De nada !
Fix made !

**Comment by adi-** [comment]
Gracias. Will you push the new version to pipy too?

**Comment by ZuluPro** [comment]
I'll look if can add other features and upload to pypi tonight.

**Comment by adi-** [comment]
I found a bug/missing functionality. Talking about postgres, there should be cmd line with `cmd += ' --dbname=???`. Currently, backup is restored to `default` DB.

**Pull request merged**